Taskfile.yml

```diff
@@ -185,6 +185,7 @@ tasks:
     cmds:
       - >
         go test -count=1 {{.RACEFLAG}} -tags={{.BUILDTAGS}} -shuffle=on -coverpkg=../...
+        -timeout=15m
```

Comment:
Do we need this change since we didn't make change to `test-integration-tigris` after all?

Reply:
We've added another test cases and the time needed to run this tests increased. This value will be also overwritten with "20" in one of existing drafts

Reply:
I'd suggest to keep it to not re-request review again 🙈 , I'll resolve the conflict in my PR

Reply:
That's fine (for now)